android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -685,6 +686,15 @@ private void initializePlayer() {
         mainHandler.postDelayed(mainRunnable, 1);
     }
 
+    public void getCurrentPosition(Promise promise) {
+        if (player != null) {
+            long currentPosition = player.getCurrentPosition() / 1000;
+            promise.resolve((double) currentPosition);
```

Comment:
Directly use double here, if you cast to long, your will loose msec information !

```suggestion
            double currentPosition = player.getCurrentPosition() / 1000;
            promise.resolve(currentPosition);
```